scrnaseq.Rmd

```diff
@@ -1942,14 +1942,23 @@ if (markers_found) {
   
   # Tests done by Enrichr
   marker_genesets_up_enriched = purrr::map(marker_genesets_up, EnrichrTest, databases=param$enrichr_dbs, padj=param$enrichr_padj)
-  
+  marker_genesets_up_enriched = purrr::map(list_names(marker_genesets_up_enriched), function(n) {
```

Comment:
Why did you need to change the above line? It was so nice and short ... was anything wrong?

Reply:
My intention was to add Database and Direction as columns. By doing this, the tables can easily be flattened into one big table. Thus we do not need to store two lists of lists of tables but only one table.

Also here the code is maybe  a bit longer but in the next chunk it is shorter and clearer.

Finally, I can attach one big table to the Seurat object.